src/contact.rs

```diff
@@ -1479,6 +1479,17 @@ mod tests {
     use crate::message::Message;
     use crate::test_utils::{self, TestContext};
 
+    #[test]
+    fn test_contact_id_values() {
+        // Some FFI users need to have the values of these fixed, how naughty.  But let's
+        // make sure we don't modify them anyway.
```

Comment:
well it is also about existing databases where it is not good if the constants are changed :)
but anyway, good to have that check - and thanks for adding it!